test/webserv/integration/test_integration.py

```
@@ -3,6 +3,8 @@
 
 import pytest
 
+from common import *
```

Ruff

`from common import *` used; unable to detect undefined names

(F403)